src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Monaco.Edit.cshtml

```diff
@@ -4,81 +4,19 @@
     var settings = Model.PartFieldDefinition.GetSettings<HtmlFieldSettings>();
     var monacoSettings = Model.PartFieldDefinition.GetSettings<HtmlFieldMonacoEditorSettings>();
     var culture = await Orchard.GetContentCultureAsync(Model.Field.ContentItem);
+    var encodedSettings = string.IsNullOrEmpty(monacoSettings.Options) ? "" : System.Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(monacoSettings.Options));
 }
 
 @await DisplayAsync(await New.ShortcodeModal())
 
 <div class="form-group">
     <label asp-for="Html">@Model.PartFieldDefinition.DisplayName()</label>
-    <div id="@Html.IdFor(x => x.Html)_editor" asp-for="Text" style="min-height: 400px;" class="form-control" dir="@culture.GetLanguageDirection()"></div>
+    <div class="form-control" dir="@culture.GetLanguageDirection()" data-monaco-oc="true" data-linked-id="@Html.IdFor(x => x.Html)" data-language="html" data-settings='@encodedSettings' data-shortcodes="true"></div>
```

Comment:
Here I see that `id="@Html.IdFor(x => x.Html)_editor"` is removed. Is it replaced by `data-linked-id="@Html.IdFor(x => x.Html)"`?